testproxy/services/bulk-mutate-rows.js

```diff
@@ -39,10 +39,17 @@ const bulkMutateRows = ({clientMap}) =>
     } catch (error) {
       if (error.name === 'PartialFailureError') {
         return {
-          status: error,
+          status: {
+            code: error.code,
+            details: [],
+            message: error.message,
+          },
           entries: Array.from(error.errors.entries()).map(([index, entry]) => ({
             index: index + 1,
-            status: entry,
+            status: {
+              code: entry.code,
+              message: entry.message,
+            },
```

Comment:
These changes are necessary because if we send information that is out of scope back to the test runner then the test runner just gets a Nil response.